Brewfile

```diff
@@ -1,4 +1,20 @@
-### Needed for BackerKit server and specs
+# About
+# This is a brewfile. It is a manifest that homebrew can use to setup your dev environment. More on brew bundle here - https://github.com/Homebrew/homebrew-bundle
+
+# To execute:
+# cd/path-to-dotfiles-repo
+# brew bundle
+
+
+# Read more about these installations in our documentation:
+# https://github.com/BackerKit/BackerKit/docs/setup_dev_environment.md
+# https://github.com/BackerKit/BackerKit/docs/setup_dev_tools.md
```

Comment:
👍  on the content. Why put it here as opposed to in the `md` files?

Reply:
I'm not sure this whole dotfiles repo is the way to go.... TBDiscussed where this stuff goes once Lindsey and I are done with the readme.